pkg/blobfuse/blobfuse.go

```diff
@@ -159,12 +159,12 @@ func appendDefaultMountOptions(mountOptions []string) []string {
 }
 
 // get storage account from secrets map
```

Comment:
pls add a return parameter comment for this function

Reply:
done.